src/Symfony/Bundle/WebServerBundle/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^5.5.9|>=7.0.8",
         "symfony/console": "~3.3",
         "symfony/http-kernel": "~3.3",
-        "symfony/process": "~3.3"
+        "symfony/process": "~3.3.14"
```

Comment:
this now forbids using 3.4, so not good. Should be `~3.3.14 || ^3.4.7`

Reply:
was patched while merged, see 743be09